ui/components/component-library/banner-tip/banner-tip.test.tsx

```diff
@@ -39,7 +45,9 @@ describe('BannerTip', () => {
     expect(getByTestId('banner-tip-greeting')).toHaveClass(
       'mm-banner-tip--logo',
     );
+    expect(getByTestId('banner-tip-greeting')).toHaveClass('custom-logo-class');
     expect(getByTestId('banner-tip-chat')).toHaveClass('mm-banner-tip--logo');
+    expect(getByTestId('banner-tip-chat')).toHaveClass('custom-logo-class');
```

Comment:
Updating test to ensure the logo className isn't overwritten when a custom class is added